src/diffusers/loaders/ip_adapter.py

```diff
@@ -138,7 +138,7 @@ def load_ip_adapter(
                 logger.info(f"loading image_encoder from {pretrained_model_name_or_path_or_dict}")
                 image_encoder = CLIPVisionModelWithProjection.from_pretrained(
                     pretrained_model_name_or_path_or_dict,
-                    subfolder=os.path.join(subfolder, "image_encoder"),
+                    subfolder=Path(subfolder, "image_encoder").as_posix(),
```

Comment:
It seems to be a good idea to follow this throughout in the library to better support Windows. 

What say @patrickvonplaten @yiyixuxu?

Reply:
Opened a tracker here: https://github.com/huggingface/diffusers/issues/6585.